src/main/java/roomescape/controller/TimeController.java

```diff
@@ -0,0 +1,57 @@
+package roomescape.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+import roomescape.domain.Time;
+import roomescape.service.TimeService;
+
+import java.net.URI;
+import java.util.List;
+
+@Controller
+public class TimeController {
+    private final TimeService timeService;
+
+    @Autowired
+    public TimeController(TimeService timeService) {
+        this.timeService = timeService;
+    }
```

Comment:
```suggestion
    public TimeController(TimeService timeService) {
        this.timeService = timeService;
    }
```
생성자 주입의 경우 `@Autowired` 어노테이션을 사용하지 않아도 됩니다!